### PR TITLE
Added SpigotMC 1.12.2

### PR DESCRIPTION
**Pull request by crock**

### CODE REVIEW

File: docsets/SpigotMC/docset.json [hunk 1]
```diff
@@ -1,13 +1,24 @@
 {
     "name": "SpigotMC",
-    "version": "1.10.2",
-    "archive": "SpigotMC.tgz",
+    "version": "1.12.2",
+    "archive": "SpigotMC-1.12.2.tgz",
```
Comment (Kapeli) [inline docsets/SpigotMC/docset.json]: You need to add this file as well.

Comment (crock) [reply docsets/SpigotMC/docset.json]: I did, I just renamed the existing version to `Spigot-1.10.2.tgz` to match the naming convention of the new version.

**Comment by Kapeli** [comment]
Your pull request doesn't contain any file at this path: `docsets/SpigotMC/SpigotMC-1.12.2.tgz`.

File: docsets/SpigotMC/docset.json (hunk 1 above)
Comment (Kapeli) [reply docsets/SpigotMC/docset.json]: Your pull request doesn't contain any file at this path: `docsets/SpigotMC/SpigotMC-1.12.2.tgz`.

Comment (crock) [reply docsets/SpigotMC/docset.json]: Fixed! :)

File: docsets/SpigotMC/docset.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "SpigotMC",
     "version": "1.12.2",
-    "archive": "SpigotMC-1.12.2.tgz",
+    "archive": "versions/1.12.2/SpigotMC-1.12.2.tgz",
```
Comment (Kapeli) [inline docsets/SpigotMC/docset.json]: This is not supported. The main docset archive needs to be separate from one in `specific_versions`.

Comment (crock) [reply docsets/SpigotMC/docset.json]: Alright fixed again! Sorry about the trouble, I don't do publish docsets very often.

**Comment by Kapeli** [comment]
Use this file for SpigotMC-1.10.2: https://kapeli.com/feeds/zzz/user_contributed/build/SpigotMC/SpigotMC.tgz.

**Comment by Kapeli** [comment]
Thank you! 👍

**Pull request merged**